test/add-approuter.spec.ts

```diff
@@ -36,9 +37,7 @@ describe('Add Approuter', () => {
 
   it('should add preconfigured files', async () => {
     const projectDir = path.resolve(pathPrefix, 'add-approuter');
-    if (fs.existsSync(projectDir)) {
-      fs.removeSync(projectDir);
-    }
+    removeDir(projectDir);
```

Comment:
This could use the same function used in other test files: `const projectDir = getCleanProjectDir(pathPrefix, 'add-approuter');`